bin/stop-dev-services

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 # stop any services running in development mode
 
-for DEBUG_PORT in 9200 9201 9202 9203
+for DEBUG_PORT in 9200 9201 9202 9203 9204 9205
```

Comment:
Just wondering if we want to bump up these ports since it's possible people might want to run Elasticsearch which defaults to 9200 and there isn't a way to change them using env vars.

Reply:
I'm fine with that. I just chose these because they are close to the one that gets used by default for debugging. Maybe start with 9500?